### ISSUE TITLE
GQL - Render Failure: Input Object type MyObject must define one or more fileds

### ISSUE THREAD

**Issue by michaelalpine** [getinsomnia/insomnia#5643]
### Expected Behavior

GQL requests should work

### Actual Behavior

I can't do anything to modify my requests and it just shows the error this feels related to https://github.com/Kong/insomnia/issues/5578

We're paying for your product and you are making it unusable. This has caused a massive time suck across my team

### Reproduction Steps

Just try and make any GQL request
![image](https://user-images.githubusercontent.com/62606815/211567491-c3005839-6b70-4dd5-9916-2c80a49f9066.png)

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

As mentioned I had a ticket open a while back where this seemed to be solved, but I got the same YC error pop up
![image](https://user-images.githubusercontent.com/62606815/211567602-3977e3f2-d622-4454-9057-95c80048205c.png)

### Insomnia Version

2022.7.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### Installation method

Downloaded from insomnia.rest

### Last Known Working Insomnia version

https://github.com/Kong/insomnia/releases/tag/core%402022.6.0

**Comment by filfreire** [comment]
Linked with #5553

**Comment by filfreire** [comment]
Hi @michaelalpine, are you still able to reproduce this in version `2022.7.1`?

**Comment by michaelalpine** [comment]
No but FYI I just realized that your automatically check for updates button must be broken. I had rolled back to 2.6.0 and unchecked that, then this happened again, and I noticed I was back on 2.7.0. That's why I thought it was still broken. But somewhere in there it updated itself when I had said not to. I've confirmed this happened with two other devs who were in the same boat. That one isn't a huge deal but would be worth double checking.
